react/features/chat/middleware.js

```diff
@@ -81,16 +79,7 @@ MiddlewareRegistry.register(store => next => action => {
         break;
 
     case OPEN_CHAT:
-        if (localParticipant.name) {
-            dispatch(setActiveModalId(CHAT_VIEW_MODAL_ID));
-            _maybeFocusField();
-        } else {
-            dispatch(openDisplayNamePrompt(() => {
```

Comment:
This seemed redundant to me, as we already display the `<DisplayNameForm />` inside the `<Chat />` component and it serves the same purpose. If it had a different reasoning, please let me know and I'll add it back.

Reply:
Looks good.

Reply:
On a second look, nope. Mobile does not have that.